.babelrc

```diff
@@ -7,7 +7,8 @@
     "transform-runtime",
     "add-module-exports",
     "transform-decorators-legacy",
-    "transform-react-display-name"
+    "transform-react-display-name",
+    "transform-class-properties"
```

Comment:
But I actually think we should just use `stage-2` instead of `stage-0`. That is stuff that is in in the pipeline to get into the standard.